swarm/util.py

```diff
@@ -119,3 +119,11 @@ def dict_slicer(dict_slice: Dict[str, Sequence]) -> Dict[str, Any]:
         raise ValueError(f"Combodict {dict_slice.keys()} misconfigured with differing lengths")
     for i in range(sizes.pop()):
         yield {k: dict_slice[k][i] for k in dict_slice}
+
+
+def func_chain(funclist: List[Callable]):
+    def inner(*args, **kwargs):
+        for f in funclist:
+            yield from f(*args, **kwargs)
+
+    return inner
```

Comment:
Not sure what this is about, also artisinal?

Reply:
This was the first pass for chaining the plot_inits and the animate functions for all the swarmplots. It turned out to be insufficient for my setup, so I had to deprecate it, but moved it to util instead. I can probably delete it.